compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -185,30 +195,31 @@ escalation "tag_resources" do
       description "Explain why you are approving the action"
     end
   end
-  run "tag_resources", data, $param_azure_tenant_id, $param_azure_sub, $param_tag_key
+  run "tag_resources", data, $param_tag_key
 end
 
-define tag_resources($data, $param_azure_tenant_id, $param_azure_sub, $param_tag_key) return $all_responses do
-
-  call get_access_token($param_azure_tenant_id) retrieve $access_token
-
+define tag_resources($data, $param_tag_key) return $all_responses do
+  $$log = []
+  $all_responses = []
+  $$subscriptionId = ""; 
+  foreach $item in $data do
+    $$subscriptionId = $item["subscriptionId"]
+  end
```

Comment:
This block doesn't look right. shouldn't the end statement happen after the tagging is completed.  Why are you setting a global variable.

Reply:
I have updated the logic as per your comments. Works fine.